sparkler-core/sparkler-deployment/docker/scripts/start.dev.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Please ensure all of your files contain an ALv2.0 header. For example see
https://github.com/USCDataScience/sparkler/blob/master/sparkler-core/bin/dockler.sh#L3-L18